lib/rubocop/cop/rails/duplicate_ignored_columns.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cops looks for assignments of `ignored_columns` that override previous
+      # assignments.
+      #
+      # Overwriting previous assignments is usually a mistake, since it will
+      # un-ignore the first set of columns
+      #
+      # @example
+      #
+      #   # bad
+      #   class User < ActiveRecord::Base
+      #     self.ignored_columns = [:one]
+      #     self.ignored_columns = [:two]
+      #   end
+      #
+      #   # bad
+      #   class User < ActiveRecord::Base
+      #     self.ignored_columns += [:one]
+      #     self.ignored_columns = [:two]
+      #   end
+      #
+      #   # good
+      #   class User < ActiveRecord::Base
+      #     self.ignored_columns = [:one, :two]
```

Comment:
Maybe not so good if:
```ruby
module IgnoreLegacy
  extend ActiveSupport::Concern
  included do
    self.ignored_columns = [:legacy]
  end
end
  
class VipUser < User
  include IgnoreLegacy

  self.ignored_columns = [:one, :two] # hello, `legacy` column!
end
```

Reply:
Indeed, but that's not something we can detect with rubocop

Reply:
Since now plain assigns are forbidden, this is no longer  problem :boom: